js/jest-tests/src/Models/BowlingGame.js

```diff
@@ -0,0 +1,52 @@
+import { TOTAL_PINS, TOTAL_FRAMES, ROLLS_PER_FRAME, STRIKE_CHAR, SPARE_CHAR, FAIL_CHAR } from "./const.js";
+
+export class BowlingGame {
+  constructor() {
+    this.rolls = [];
+  }
+
+  roll(pins) {
+    this.rolls.push(pins);
+  }
+
+  get score() {
```

Comment:
Hai un bo equilibrio entre Game Frame e Score, aínda que estaría ben ter unha clase Frame, asumindo que imos seguir traballando en Bowling, onde Frame é un concepto de primer nivel só por debaixo de Game

Reply:
Orixinalmente tiña unha implementación para Roll, Frame e Line, que tiña a lóxica separada do xogo, ainda que Roll era máis un Value Object. Pero tiven uns problemas e non din feito que a implementación me funcionase correctamente por culpa do último frame. Decidin facer algo máis simple para poder entregalo antes de rematar o curso